meeting-minutes/client-minutes/nov10-client-minutes.md

```diff
@@ -0,0 +1,83 @@
+  - **Mentor Forms**
+    - Name, email, list of ranked skills, text-area to obtain a description from the user about themselves/their goals for the program, some way of obtaining the user's availability, and password for account creation.
+  - NOTE: After discussing what input fields are necessary on each form, the client was contemplating whether we should be collecting the user's time-zone, or if collecting the user's location would suffice. She mentioned that the time-zone could possibly be extracted from the user's browser, but location alone might suffice.
+- A road block from this week was the complexity that matching based on common availability introduces. Is common availability **required** when it comes to making mentee and mentor matches, or can we match primarily based on skills and time-zones?
+  - The client recognizes the complexity that matching based on common availability might have. As such, when creating mentee and mentor matches, **common availability does not need to be considered**.
+    - The client's recommended course of action: The process can be analogous to a job recruiting process (i.e The individual conducting interviews sends out a calendar to the interviewee. The interviewee receives a calendar of available time slots, and picks one). Therefore, for our project we can:
+      - Obtain availability for Mentors only.
+      - Once a match has been made, send the Mentor's availability to the Mentee.
+      - The mentee can then select a timeslot based on when the Mentor is available.
+
+The client asked how our team was progressing with our use of Firebase, so the MentorshipForm.js file was shown since authentication was worked on over the last sprint. The client recommended also initializing Firebase within App.js as well (so that all the components being called from App.js have access to the information in the database).
+
+The client also mentioned that we should keep in mind that the **Mentee** form will be embedded onto a public website, so we should consider how it will be turned into an iFrame tag.
+
+### ACTION ITEMS
+
+11/10-01. Re-vamp the Mentee and Mentor forms to include the input fields as listed. Since Eric was working on separating the MentorshipForm.js into a form for Mentees in specific, and another for Mentors, he could potentially refine the fields for each. **Should be completed by the end of this week's sprint [Monday]**
+11/10-02. The team should decide whether we want to remove the time-zone dropdown and replace it with obtaining a user's location. Potentially, we could keep the time-zone dropdown and add a field to obtain the user's location. We could present this during next week's client meeting, and inform the client of associated pros and cons to determine if we replace time-zone with user location. **This should be discussed with the team, and decided on by Wednesday so we can incorporate changes if necessary**
```

Comment:
Technically, I know how to and can quickly implement user localization querying (it's very simple using [this](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API)), but I'm against it, UI-wise. There's this one website that keeps forcing me to use their french version, since I'm in France, when I am trying to access the default (US) version, and it's very annoying. 

I suggest we leave the dropdown as it is. As an enhancement, in the future, we can auto-fill it based on user's location, all the while letting the user have the possibility to change it if they want (not hard to do). 

If the client feels strongly about doing otherwise, her decision overrides my opinion, of course.

Reply:
I agree with keeping the drop-down for time-zone selection. To me, this makes more sense when it comes to matching potential mentees and mentors, since it's slightly less restrictive. I feel like if we were to limit users based on their location (i.e Mentees in Kelowna only match with Mentors in Kelowna), it would restrict some potentially great matches (especially in the context of COVID where the majority of people are working remotely), whereas time-zones keep things slightly more open. 
To show the client that we are considerate of her wishes, I think we should probably add an input field of some sort (such as City and Country) to show for next week.

Reply:
This is fair, but unless the client puts a big importance on the location field, I think we should postpone it. If we keep making changes (and this one is not trivial, getting location isn't as easy as just getting a string), we'll have a hard time building an MVP before peer testing #1. I'd personally rather show an almost finished MVP next week than one where extra fields have been added. Does Shannon feel very strongly about this? If so, sure, we can do it. From your notes, it sounded like this was just something she was considering, as an enhancement. @KatrinaMart

Reply:
@jpabadir From my conversation with the client, the idea of a location field was an after-thought. Since the peer testing milestone is fast approaching, I notified the client during the meeting that we're currently working towards that deadline. So, I'll let her know that we've been prioritizing building a MVP, but that we can revisit location in the future if that is what she'd prefer (presenting the cons that we've discussed).